doc/how-tos/export-data-out-pachyderm.md

```diff
@@ -0,0 +1,163 @@
+# Export Your Data From Pachyderm
+
+After you build a pipeline, you probably want to see the
+results that the pipeline has produced. Every commit into an
+input repository results in a corresponding commit into an
+output repository.
+
+To access the results of
+a pipeline, you can use one of the following methods:
+
+* By running the `pachctl get file` command. This
+command returns the contents of the specified file.<br>
+To get the list of files in a repo, you should first
+run the `pachctl list file` command.
+See [Export Your Data with `pachctl`](#export-your-data-with-pachctl).<br>
+
+* By configuring the pipeline. A pipeline can push or expose
+output data to external sources. You can configure the following
+data exporting methods in a Pachyderm pipeline:
+
+  * An `egress` property enables you to export your data to
+  an external datastore, such as Amazon S3,
+  Google Cloud Storage, and others.<br>
+  See [Export data by using `egress`](#export-your-data-with-egress).<br>
+
+  * A service. A Pachyderm service exposes the results of the
+  pipeline processing on a specific port in the form of a dashboard
+  or similar endpoint.<br>
+  See [Service](../concepts/pipeline-concepts/pipeline/service.html).<br>
+
```

Comment:
There's a 3rd option here too:

* Since a pipeline is just a docker container running your code, you can egress data to any data source (even those not officially supported by Pachyderm's `egress` field), by just connecting to that source from within your code.

Reply:
added.